index.html

```diff
@@ -63,7 +65,63 @@ <h1>I'm Tesfaye <br>Glad to see<br> you!</h1>
         </ul>
       </section>
     </main>
-<div class="card-container">
+    
+     <!--Project section-->
+
+     <div id="main-pop" class="hidden">
+      <div>
+        <div id="main-pop-head">
+          <h2>Multi-Post Stories</h2>
+          <span>&#x2715;</span>
+        </div>
+
+        <ul>
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
```

Comment:
- [Optional] Good job making the changes to the modal @eyoab01, But I notice the modal has a scroll at the bottom, and users are required to scroll only the bottom section to see the buttons. This happens on both the mobile and desktop modals. Please remember this is not good practice when it comes to the UX, Also, doing this will help you align your design even better with the figma design. 
![Screenshot 2023-02-10 205910](https://user-images.githubusercontent.com/69077061/218164022-d3261384-22da-4099-a1f2-9c83d054d191.png)